tuning/trackers/filelogging_tracker.py

```diff
@@ -56,7 +56,9 @@ def _track_loss(self, loss_key, log_name, log_file, logs, state):
             log_obj = {
                 "name": log_name,
                 "data": {
-                    "epoch": round(logs["epoch"], 2),
+                    "epoch": round((logs["epoch"] - 1), 2)
+                    if logs["epoch"] != 1.0
+                    else 0,
```

Comment:
For consistency, let's do "0.0" instead of "0".

Reply:
Okay Sure. I can remove the conversion from 0.0 to 0

Reply:
Also regarding writing unit case for the same, I can think of is to read the log file and check the first line of the log file. Does this sound good to you for a base logic of unit test case ?

Reply:
yea that sounds good!